Libraries/NativeModules/specs/NativeDevSettings.js

```diff
@@ -28,8 +28,10 @@ export interface Spec extends TurboModule {
   // iOS only.
   +setIsShakeToShowDevMenuEnabled: (enabled: boolean) => void;
 
+  // [macOS
   // macOS only.
-  +setIsSecondaryClickToShowDevMenuEnabled: (enabled: boolean) => void; // TODO(macOS GH#774)
+  +setIsSecondaryClickToShowDevMenuEnabled: (enabled: boolean) => void;
+  // macOS]
```

Comment:
optional can we just remove the "macOS only" comment? and just have a single line tag // [macOS] at the end of the line

Reply:
I prefer we keep it to match the `// iOS only` comment above.